frontend/public/components/sidebars/network-policy-sidebar.jsx

```diff
@@ -16,54 +18,61 @@ const samples = [
     img: denyOtherNamespacesImg,
     details: 'Deny traffic from other namespaces while allowing all traffic from the namespaces the Pod is living in.',
     templateName: 'deny-other-namespaces',
+    kind: referenceForModel(NetworkPolicyModel),
```

Comment:
I swear I added these...

Reply:
NP at all @alecmerdler !!! the more we fix the more we learn ! :)

Reply:
Actually I really like how the YAML editor with its bits and pieces works :+1:

Reply:
Have you checked out the YAML editor autocompletion?